File: pinyougou_parent/pinyougou_manager_web/src/main/webapp/js/service/specificationService.js
app.service("specificationService",function($http){
	
	this.findSpecList=function(){
		return $http.get("../specification/findSpecList");
	}
	
	 this.findPage=function(pageNo,pageSize){
		 return $http.get("../specification/findPage/"+pageNo+"/"+pageSize);
	 }
	 
	 this.findAll=function(){
		 return $http.get("../specification/findAll");
	 }
	 
	 
	 this.add=function(entity){
		 return $http.post("../specification/add",entity);
	 }
	 this.update=function(entity){
		 return  $http.post("../specification/update",entity);
	 }
	 
	 
	 this.findOne=function(id){
		 return $http.get("../specification/findOne/"+id);
	 }
	 
	 this.dele=function(selectIds){
		 return $http.get("../specification/dele/"+selectIds);
	 }
	 
	 this.search=function(pageNo,pageSize,searchEntity){
 		return $http.post("../specification/search/"+pageNo+"/"+pageSize,searchEntity);
 	 }
	
})

File: pinyougou_parent/pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/SpecificationService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbSpecification;

import entity.PageResult;
import groupEntity.Specification;

public interface SpecificationService {
	
	public List<TbSpecification> findAll();

	public PageResult findPage(int pageNo, int pageSize);

	public void add(Specification specification);

	public Specification findOne(Long id);

	public void update(Specification specification);

	public void dele(Long[] ids);

	public PageResult findPage(int pageNo, int pageSize, TbSpecification specification);

	public List<Map> findSpecList();
	
}
